pkg/genevalogging/genevalogging.go

```diff
@@ -154,107 +153,26 @@ type genevaLogging struct {
 	log *logrus.Entry
 	env env.Interface
 
-	oc *api.OpenShiftCluster
-
-	cli    kubernetes.Interface
-	seccli securityclient.Interface
+	oc      *api.OpenShiftCluster
+	ensurer ensure.Interface
 }
 
-func New(log *logrus.Entry, e env.Interface, oc *api.OpenShiftCluster, cli kubernetes.Interface, seccli securityclient.Interface) GenevaLogging {
+func New(log *logrus.Entry, e env.Interface, oc *api.OpenShiftCluster, ensure ensure.Interface) GenevaLogging {
```

Comment:
```suggestion
func New(log *logrus.Entry, e env.Interface, oc *api.OpenShiftCluster, ensurer ensure.Interface) GenevaLogging {
```

(call it ensurer everywhere)